githubscript/examples/task.example.json

```
@@ -17,8 +17,8 @@
   "deadline": "2020-06-17T22:32:06.776Z",
   "expires": "2021-06-16T22:32:06.776Z",
   "scopes": [
-    "project:mobile:fenix:releng:github:project:fenix",
-    "project:mobile:fenix:releng:github:action:release"
+    "project:mobile:firefox-android:releng:github:project:fenix",
```

Good idea on updating these scopes! I hadn't thought of that. 

Quick nit:

```suggestion
    "project:mobile:firefox-android:releng:github:project:firefox-android",
```

Per https://firefox-ci-tc.services.mozilla.com/tasks/WmaXtdzUT8Kk0p822Qubgg

---

Good catch - thanks.